test/plugin/TokenReceiverPlugin.t.sol

```diff
@@ -75,12 +75,24 @@ contract TokenReceiverPluginTest is OptimizedTest, IERC1155Receiver {
     }
 
     function test_failERC1155Transfer() public {
-        // for 1155, reverts are caught in a try catch and bubbled up with a diff reason
-        vm.expectRevert("ERC1155: transfer to non-ERC1155Receiver implementer");
+        // for 1155, reverts are caught in a try catch and bubbled up with the reason from the account
```

Comment:
Is this a change in the OZ ERC-1155 token contract?

Reply:
yep